src/CoreWCF.Primitives/src/CoreWCF/Description/DispatcherBuilder.cs

```diff
@@ -590,6 +593,16 @@ private static void BuildDispatchOperation(OperationDescription operation, Dispa
 
                 parent.UnhandledDispatchOperation = child;
             }
+
+            var authorizationPolicyProvider = services.GetService<IAuthorizationPolicyProvider>();
+            if (authorizationPolicyProvider != null)
+            {
+                // TODO: Make this chain call async
+                Task<AuthorizationPolicy> getPolicyTask = operation.AuthorizeData.Count > 0
+                    ? AuthorizationPolicy.CombineAsync(authorizationPolicyProvider, operation.AuthorizeData)
+                    : authorizationPolicyProvider.GetDefaultPolicyAsync();
```

Comment:
@mconnew please notice that ASP.NET Core 3 introduced a `FallbackPolicy` which differs of the DefaultPolicy.
- `DefaultPolicy` is applied when `AuthorizeData` is applied without specifying a policy name.
- `FallbackPolicy` is applied when no Authorize attribute is applied.

The behavior implemented here is that we apply the `DefaultPolicy` when no `AuthorizeData` is provided.